src/goto-programs/goto_program_template.h

```diff
@@ -265,6 +265,23 @@ class goto_program_templatet
     return t;
   }
 
+  inline static const irep_idt get_function_id(
```

Comment:
I don't think "inline" is necessary here (the definition is within the class definition).